config/src/main/java/org/hyperledger/besu/config/QbftConfigOptions.java

```diff
@@ -23,4 +24,8 @@
   default boolean isValidatorContractMode() {
     return getValidatorContractAddress().isPresent();
   }
+
+  default OptionalLong getStartBlock() {
```

Comment:
Don't we think should have a default implementation that does nothing. It only makes sense for the MutableQbftConfigOptions to have an empty value.